packages/framework/build.xml

```diff
@@ -873,6 +880,21 @@
         </exec>
     </target>
 
+    <target name="migrate-uploaded-files" description="Move all uploaded files from subdirectories to the top level directory." hidden="true">
```

Comment:
it seems like this target is never used, am I right? Is it necessary for something? For testing purposes? Maybe it could be mentioned in the upgrade notes - something like "you can debug the command locally using `migrate-uploaded-files` phing target"? I am not sure